core/src/main/scala/kafka/coordinator/group/GroupMetadataManager.scala

```diff
@@ -1378,6 +1350,28 @@ object GroupMetadataManager {
       "%X".format(BigInt(1, bytes))
   }
 
+  def maybeConvertError(error: Errors) : Errors = {
+    error match {
+      case Errors.UNKNOWN_TOPIC_OR_PARTITION
+           | Errors.NOT_ENOUGH_REPLICAS
+           | Errors.NOT_ENOUGH_REPLICAS_AFTER_APPEND =>
+        Errors.COORDINATOR_NOT_AVAILABLE
+
+      case Errors.NOT_LEADER_OR_FOLLOWER
+           | Errors.KAFKA_STORAGE_ERROR =>
+        Errors.NOT_COORDINATOR
+
+      case Errors.MESSAGE_TOO_LARGE
+           | Errors.RECORD_LIST_TOO_LARGE
+           | Errors.INVALID_FETCH_SIZE =>
+        Errors.INVALID_COMMIT_OFFSET_SIZE
```

Comment:
Wait, why did we change this?

Reply:
I pulled from storeGroup instead of storeOffsets 🤦‍♀️ i will fix this tomorrow